app/render/renderer.cpp

```diff
@@ -240,9 +240,17 @@ bool Renderer::GetColorContext(const ColorTransformJob &color_job, Renderer::Col
       const char* sampler_name = nullptr;
       unsigned int width = 0, height = 0;
       OCIO::GpuShaderDesc::TextureType channel = OCIO::GpuShaderDesc::TEXTURE_RGB_CHANNEL;
+#if OCIO_VERSION_HEX >= 0x02030000
+      OCIO::GpuShaderDesc::TextureDimensions dimensions = OCIO::GpuShaderDesc::TEXTURE_2D;
+#endif
       OCIO::Interpolation interpolation = OCIO::INTERP_LINEAR;
 
-      shader_desc->getTexture(i, tex_name, sampler_name, width, height, channel, interpolation);
+      shader_desc->getTexture(i, tex_name, sampler_name, width, height, channel,
```

Comment:
Seems a bit out of line with the coding style to break off the line for the preprocessor conditional.
I'd rather see two instances of the call, but maybe it's just my personal taste ;)